arch/csr/Smcntrpmf/mcyclecfg.yaml

```diff
@@ -0,0 +1,78 @@
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+$schema: csr_schema.json#
+kind: csr
+name: mcyclecfg
+long_name: Machine Cycle Counter Configuration
+address: 0x321
+priv_mode: M
+length: 64
+base: 64
```

Comment:
Reading closer, this register is _available_ for RV32, but there are no interesting bits in that mode, since it's the lower 32 bits only, and there are no fields in the lower 32 bits. Regardless, it is defined, so remove "base: 64".